sysid-application/src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -323,6 +359,18 @@ void Analyzer::PrepareRawGraphs() {
   });
 }
 
+void Analyzer::PrepareGraphs() {
+  WPI_INFO(m_logger, "{}", "Graph state");
+  AbortDataPrep();
+  m_dataThread = std::thread([&] {
+    m_plot.SetData(m_manager->GetRawData(), m_manager->GetFilteredData(),
+                   m_manager->GetUnit(), m_ff, m_manager->GetStartTimes(),
+                   m_manager->GetAnalysisType(), m_abortDataPrep);
+  });
+  UpdateFeedbackGains();
```

Comment:
Actually I see the issue, you call `UpdateFeedbackGains` inside `PrepareGraphs` in which `UpdateFeedbackGains` calls `PrepareGraphs` causing an infinite loop of graph preparation.

Reply:
I don't think this is true?  I don't see any call to `PrepareGraphs` from `UpdateFeedbackGains`, and it doesn't crash on my machine.

Reply:
Oops yeah I misread